.github/workflows/test.yml

```diff
@@ -1,20 +1,20 @@
 name: Test
 on:
   pull_request:
-    branches:
```

Comment:
What is the effect of this change? Is it that pull request to ANY branch will trigger build test now?

Reply:
Yes, it's so contributors can test our changes on our forks _before_ creating PRs.

Otherwise we have to develop on `master` (bad) or open PRs without fully testing and hoping for the best (also bad).

Reply:
Or rather: my answer was about lines 7-8 below.

Yes, 4-5 would also allow testing PRs to any branch as well.

Not as important as 7-8, but I see occasionally see it used by projects: 

A contributor opens a big PR, reviewer makes a PR to the contributor's branch because there's more changes than can be made in the review UI.